graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/EntitiesClientApiGenTest.kt

```diff
@@ -114,7 +114,7 @@ class EntitiesClientApiGenTest {
                 name: String @external
                 age: Int
             }
-            
+
```

Comment:
Why all these whitespace changes?

Reply:
Good point, when initially working on this I moved through the files using Vim and as I have it configured it removes lines with just whitespaces automatically.